spackbot/comments.py

```diff
@@ -3,17 +3,23 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import io
 import random
+import traceback
 import spackbot.helpers as helpers
 
 
 async def tell_joke(gh):
     """
     Tell a joke to ease the PR tension!
     """
-    joke = await gh.getitem(
-        "https://official-joke-api.appspot.com/jokes/programming/random"
-    )
+    try:
+        joke = await gh.getitem(
+            "https://official-joke-api.appspot.com/jokes/programming/random"
+        )
+    except Exception:
+        return "To be honest, I haven't heard any good jokes lately."
```

Comment:
I like it.

Reply:
I like the idea too but every time I try the site I get a `Server Error`.

Reply:
The jokes were already there (thanks to Vanessa). I just meant the exception response.

Reply:
I think both are great.  I was commenting on the overall feature and the exception.